server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/AbstractRecipientRewriteTableMailet.java

```diff
@@ -19,22 +19,28 @@
 

```

Comment:
Couldn't we get ride of this abstract class ? (If some code is shared via heritage, please share it via composition)